src/example-button.module/meta.json

```diff
@@ -0,0 +1,7 @@
+{
+  "label": "Example button",
+  "host_template_types": ["PAGE", "BLOG_LISTING", "BLOG_POST"],
+  "icon": "",
+  "master_language": "en",
+  "is_available_for_new_content": true
+}
```

Comment:
This is missing categories

Reply:
Just added this in (picked my best guess at category from the descriptions) but feel free to adjust it if there is a better category for this particular module.